.circleci/config.yml

```diff
@@ -292,15 +302,17 @@ jobs:
           destination: node3/
 
   docker_image_build:
```

Comment:
Maybe name shall be updated from docker_image_build as this does much more than build?